backend/chainlit/server.py

```diff
@@ -255,61 +258,22 @@ def replace_between_tags(text: str, start_tag: str, end_tag: str, replacement: s
     return re.sub(pattern, start_tag + replacement + end_tag, text, flags=re.DOTALL)
 
 
-def get_html_template():
-    PLACEHOLDER = "<!-- TAG INJECTION PLACEHOLDER -->"
-    JS_PLACEHOLDER = "<!-- JS INJECTION PLACEHOLDER -->"
-    CSS_PLACEHOLDER = "<!-- CSS INJECTION PLACEHOLDER -->"
-
-    default_url = "https://github.com/Chainlit/chainlit"
-    default_meta_image_url = (
-        "https://chainlit-cloud.s3.eu-west-3.amazonaws.com/logo/chainlit_banner.png"
-    )
-    url = config.ui.github or default_url
+def get_html_template(request: Request):
+    default_url = "https://app.purposepilot.co/"
+    default_meta_image_url = "https://app.purposepilot.co/logo?theme=light"
+    url = config.ui.custom_url or default_url
     meta_image_url = config.ui.custom_meta_image_url or default_meta_image_url
     favicon_path = ROOT_PATH + "/favicon" if ROOT_PATH else "/favicon"
 
-    tags = f"""<title>{config.ui.name}</title>
-    <link rel="icon" href="{favicon_path}" />
-    <meta name="description" content="{config.ui.description}">
-    <meta property="og:type" content="website">
-    <meta property="og:title" content="{config.ui.name}">
-    <meta property="og:description" content="{config.ui.description}">
-    <meta property="og:image" content="{meta_image_url}">
```

Comment:
I would not call this `meta_image_url` because this is the [OpenGraph](https://ogp.me/) standard. Which is used by much more than just Meta.

In addition, we will have to support [Twitter Cards](https://developer.x.com/en/docs/twitter-for-websites/cards/overview/abouts-cards). But note that all of these headers will be specified _in verbatim_ in the configuration file, once we've specified the appropriate header.

Reply:
I feel silly, I see now that it was not you who put this but it was `meta_image_url` already in chainlit. I feel a bit silly, but in the context of compatibility, I would suggest not to change this.